lms/templates/courseware/courses.html

```diff
@@ -45,9 +57,19 @@
   </header>
 
   <section class="courses-container">
+
+    <div id="discovery-search">
+      <form>
+        <input placeholder="${_('Search for a course')}" type="text"/>
+        <button type="submit" class="button postfix" aria-label="${_('Search')}">
+          <i class="icon fa fa-search" aria-hidden="true"></i>
+        </button>
+      </form>
+    </div>
+
     <section class="courses">
       <ul class="courses-listing">
-	%for course in courses:
+	      %for course in courses:
```

Comment:
Nice catch! About the 2 spaces, I see it a lot in html templates. I just follow whatever format I find in the file I'm editing.